src/components/Expense/index.jsx

```diff
@@ -102,6 +126,32 @@ const Expense = () => {
         </div>
       </footer>
 
+      <div className="dashboard_expense">
+        <div className="dashboard_expense_left">
+          <h2>YOU OWE</h2>
+          <div>
+            {getOweLedgerOfCurrentUser().map(obj => 
+              <div>
+                <h3>
+                  $ {obj.borrowed_amount} to {getUserById(obj.payer_id).name} ({getExpenseById(obj.expense_id).description})
```

Comment:
Heavy yar, dollers my chal rahy ho ap to pakistan my rah kar.
kamal isi iya to itna ameer hy